scripts/agent_dataset_stats.py

```
@@ -0,0 +1,82 @@
+Example:
+    python scripts/agent_dataset_stats.py ReceiptsTable-dc5be22 dataset_stats.json
+"""
+
+from receipt_dynamo import DynamoClient
+from receipt_layoutlm.receipt_layoutlm.data_loader import CORE_LABELS
+import json
+import sys
+from typing import Dict, Any
+
+
+def get_dataset_stats(table_name: str, output_file: str = None) -> Dict[str, Any]:
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)